text/1946-intra-rustdoc-links.md

```diff
@@ -260,6 +260,34 @@ fn foo2() {
 }
 ```
 
+### Cross-crate re-exports
+
+If an item is re-exported from an inner crate to an outer crate,
+its documentation will be resolved the same in both crates, as if it were in
+the original scope. For example, this function will link to `f` in both crates,
+even though `f` is not in scope in the outer crate:
+
+```rust
+// inner-crate
+
+pub fn f() {}
+/// This links to [f].
+pub fn g() {}
+```
+
+```rust
+// outer-crate
+pub use inner_crate::g;
+```
+
+### Links to private items
+
+If a public item links to a private one, and `--document-private-items` is not passed,
+rustdoc should give an error. If a private item links to another private
```

Comment:
This isn't currently an error. It triggers the `broken_intra_doc_links` lint which is warn by default. The same applies to the other errors.

Reply:
I said 'error' to be consistent with the existing RFC: https://github.com/rust-lang/rfcs/blob/master/text/1946-intra-rustdoc-links.md#path-ambiguities

> Rustdoc will throw an error if you use a non-disambiguated path in the case of there being a value in both the type and value namespace.

Should I change that to say 'warning' as well? The reason I think 'error' describes it better is because rustdoc won't generate the link in those cases.